exporter/awsemfexporter/metric_translator.go

```diff
@@ -118,13 +131,9 @@ func (dps DoubleHistogramDataPointSlice) At(i int) DataPoint {
 	return dps.DoubleHistogramDataPointSlice.At(i)
 }
 
-// TranslateOtToCWMetric converts OT metrics to CloudWatch Metric format
-func TranslateOtToCWMetric(rm *pdata.ResourceMetrics, config *Config) ([]*CWMetrics, int) {
-	var cwMetricList []*CWMetrics
-	totalDroppedMetrics := 0
-	namespace := config.Namespace
-	var instrumentationLibName string
 
+// Validates namespace for given set of metrics from user config 
+func validateNamespace(rm *pdata.ResourceMetrics, namespace string) (string) {
```

Comment:
Don't think we're _validating_ the namespace here. It looks to me like we're _retrieving_ it. So, something like `retrieveNamespace` or `parseNamespace` might be a better fit.